manifests/cf-manifest/spec/manifest/env_spec.rb

```diff
@@ -41,14 +41,15 @@ def get_instance_group_instances(manifest, instance_group_name)
       end
 
       describe "doppler" do
-        it "instance count should be at least 1:2 with cell count" do
+        it "instance count should be at least half of the cell count" do
           doppler_instance_count = env_manifest.fetch("instance_groups.doppler").dig("instances").to_f
           cell_instance_count = env_manifest.fetch("instance_groups.diego-cell").dig("instances").to_f
 
-          ratio = cell_instance_count / doppler_instance_count
+          half = cell_instance_count / 2
+          half_with_headroom = half + 3
```

Comment:
I think this should be half + AZ count

Reply:
Or maybe (half + AZ count) rounded to the nearest multiple of AZ count

Reply:
Yeah, you're right!

Reply:
Added round_up :)